packages/lib/src/vis/surface/SurfaceVis.tsx

```diff
@@ -22,7 +23,7 @@ interface Props {
   showPoints?: boolean;
 }
 
-function SurfaceVis(props: PropsWithChildren<Props>) {
+function SurfaceVis(props: PropsWithChildren<Props> & ClassStyleAttrs) {
```

Comment:
This visualization's `Props` interface is used in `SurfaceMesh`, so I could not extend it. I went with this syntax instead.

Reply:
Perhaps `SurfaceVisProps` should be in a `models` file then to be extended/imported in both components :shrug:

Reply:
In the end, I created a dedicated `Props` interface in `SurfaceMesh`.